joshprzybyszewski/cribbage
# add a link to the wasm app

## What broke / What you're adding
The WASM client is not discoverable. Until our react client works well, we should have an easy link to get to it.

## How you did it
I added a WASM png from [their site](https://github.com/WebAssembly/web-assembly-logo) (creative commons license nbd) and then put it next to our home button. This should be obvious enough.

## How to test it and how to try to break it
Run the app (by pulling this PR's docker image and running that, otherwise the dev client doesn't work cuz it's on port 3000). Click the wasm png. Get taken to `/wasm` which should access the wasm client.
